okhttp/src/main/java/okhttp3/internal/DiskLruCache.java

```diff
@@ -151,6 +151,7 @@
   // Must be read and written when synchronized on 'this'.
   private boolean initialized;
   private boolean closed;
+  private boolean preventWrites;
```

Comment:
I wonder if there’s a better name for this . . . later we might have multiple conditions and this will be just one of them. Maybe rename to `mostRecentTrimFailed` ?